homeassistant/components/blink/services.py

```diff
@@ -86,23 +89,30 @@ async def async_handle_save_video_service(call: ServiceCall) -> None:
         video_path = call.data[CONF_FILENAME]
         if not hass.config.is_allowed_path(video_path):
             _LOGGER.error("Can't write %s, no access to path!", video_path)
-            return
-        for coordinator in await collect_coordinators(call.data[ATTR_DEVICE_ID]):
+            raise ServiceValidationError(
```

Comment:
Maybe add translation keys as well, while we are touching this now anyways?

Reply:
Needs to happen at some point, Why not now.